mogc_partnerships/patches.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def patch_skip_activation_email():
+    try:
+        from openedx.core.djangoapps.user_authn.views import register
+    except ImportError:
+        return
+
+    from .models import CohortMembership
```

Comment:
I'm pretty sure this will work, but I'm making this comment that we should be on the lookout for it. I'm probably scarred from Python 2 days where importing modules was not as consistent as it is in Python 3 (and 3.4+).